### DISCUSSION TITLE
Sonoff Zigbee 3.0 Dongles Advice?

### DISCUSSION THREAD

**Discussion by ghubg10001** [Koenkk/zigbee2mqtt#19476] [Devices]
I ordered a Sonoff Zigbee 3.0 dongle off of Walmart trying to get MQTT and Zigbee2MQTT working. Having all sorts of issues. I am beginning to wonder about versions of the stick floating around. Anyone have any input or useful information to understand?

The one I have is the Sonoff Zigbee 3.0 USB Dongle Plus v2 which I believe is the ZBDongle-P. But now I am reading and it sounds like that one can be unstable and even on the Zigbee2MQTT website I should be using the ZBDongle-E. 

Again just looking for some input. I am a relatively casual smart home tinkerer. For years I had success with the Deconz zigbee dongle and Homebridge. But then I got into wanting to try out continuing to use Homebridge and setup docker and portioner on a new Pi with MQTT and Zigbee2MQTT and it sounded like the Sonoff device was better for doing this. While it seems I can get the MQTT server running OK I have a really difficult time getting Zigbee2MQTT going and it either only shows under ttyACM0 vs ttyUSB0 or doesn't list a connection at all and its really been frustrating.

**Comment by Koenkk** [comment]
Dongle P = fully supported, Dongle E = experimental

**Reply by ghubg10001** [reply]
Appreciate the comment. Looking at the back of the device it is ZBDongle-E. I’ll go ahead and give ZBDongle-P a shot. Super confusing.

**Reply by kwetnico** [reply]
Hello, when do you think the ZBDongle-E will become compatible and more experimental, many of us want to switch to it? (I currently have a conbee2 but I would like to upgrade)
Thanks
